bfm/tests/CMakeLists.txt

```diff
@@ -15,25 +15,48 @@
 # You should have received a copy of the GNU Lesser General Public
 # License along with this library; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+cmake_minimum_required(VERSION 3.6)
+project(bfm_test C CXX)
+include(${BF_SCRIPTS_DIR}/cmake/flags/flags.cmake)
+include(CTest)
+enable_testing(true)
+enable_language(ASM_NASM)
```

Comment:
Same thing as above for both nasm and testing. Please search for all instances and clean this up.